Susi.xcodeproj/project.pbxproj

```diff
@@ -764,7 +764,7 @@
 					};
 					6D4D9B361E768E190067459C = {
 						CreatedOnToolsVersion = 8.2.1;
-						DevelopmentTeam = 7P4K4M8A4Q;
+						DevelopmentTeam = 29HRCWU889;
```

Comment:
Changes to this file aren't required.

Reply:
could not build it without it for testing due to profile error.